test/cmocka/src/math/arithmetic/exponential.c

```diff
@@ -65,7 +66,7 @@ static void test_math_arithmetic_exponential_fixed(void **state)
 	int32_t b_i;
 
 	srand((unsigned int)time(NULL));
-	for (i = 0; i < 256; i++) {
+	for (i = 0; i < 1024; i++) {
```

Comment:
This unrelated change could be in another commit with title indicating improvement of cmocka test coverage. I assume the old test with 256 points would work with hifi4 version too.